### ISSUE TITLE
[hip-runtime-amd/nvidia] Hyphens in library names

### ISSUE THREAD

**Issue by tpkessler** [rocm-arch/rocm-arch#684]
Compilation with `hip` fails because of unnecessary hyphens at the end of the library names,
```bash
ls /opt/rocm/hip/lib
cmake           libamdhip64.so.5           libhiprtc-builtins.so    libhiprtc-builtins.so.5.0.13601-
libamdhip64.so  libamdhip64.so.5.0.13601-  libhiprtc-builtins.so.5
```
Now, `clang` expects input from stdin.

**Comment by acxz** [comment]
patched with 045cd7fa2aad7872cf77816f6accd372e858b410

**Comment by tpkessler** [comment]
Fixed with 5.3.0
